src/hardware.cpp

```diff
@@ -131,6 +131,67 @@ fba::Hardware::Hardware(std::string const & timestr,
         loc_phi_max[location[i]] = phi_max[i] * M_PI / 180.0;
         loc_phi_pos[location[i]] = phi_pos[i] * M_PI / 180.0;
         loc_phi_arm[location[i]] = phi_arm[i];
+
+        if (state[location[i]] & (FIBER_STATE_STUCK | FIBER_STATE_BROKEN)) {
+            if ((loc_theta_pos[location[i]] < loc_theta_min[location[i]]) ||
+                (loc_theta_pos[location[i]] > loc_theta_max[location[i]])) {
+                logmsg.str("");
+                logmsg << "STUCK/BROKEN positioner (loc " << location[i]
+                       << ") theta value is outside of range: theta = "
+                       << loc_theta_pos[location[i]] << " vs range [" << loc_theta_min[location[i]]
+                       << ", " << loc_theta_max[location[i]] << "]; CLIPPING to valid range";
```

Comment:
We talked about this offline...  Basically this clipping could result in an output RA/DEC of the stuck positioners that did not match the corresponding POS_T / POS_P from the online DB (where those angles may slightly exceed the "targetable" range we use in fiberassign.